src/plugins/plugin-grpc.ts

```diff
@@ -108,6 +108,24 @@ function unpatchMetadata() {
 }
 
 function patchClient(client: ClientModule, api: TraceAgent) {
+  /**
+   * Set trace context on a Metadata object if it exists.
+   * @param metadata The Metadata object to which a trace context should be
+   * added.
+   * @param span The span that contains the trace context to add as a metadata
+   * entry.
+   */
+  function setTraceContextFromString(metadata: Metadata, span: Span): void {
```

Comment:
nit: based on the name, I would have expected the second parameter to be a string. The only thin we do with `span` in this function is to call `getTraceContext` on it. Could the caller do this call?

Reply:
Makes sense. The way you said also mirrors the corresponding function on the server side better.